core/src/main/java/io/confluent/rest/alpn/server/BouncyCastleServerALPNProcessor.java

```diff
@@ -0,0 +1,110 @@
+/*
```

Comment:
If this is copied from somewhere, I think we need to credit the original source.  And also check the license on that original source file to ensure it's not eg GPL.

Reply:
it is Eclipse Public License v1.0 https://www.tldrlegal.com/license/eclipse-public-license-1-0-epl-1-0, similar to GPL, so a bit more complicated.

Reply:
changed to use class inheritance method, so there is no license concern.